wcfsetup/install/files/lib/core.functions.php

```diff
@@ -140,6 +140,40 @@ function logThrowable(\Throwable $e, &$logFile = null): string {
 			return str_replace("\n", ' ', $item);
 		};
 		
+		$getExtraInformation = function (\Throwable $e) {
+			$extraInformation = [];
+			$contextLineCount = 5;
+			
+			if ($e instanceof IExtraInformationException) {
+				$extraInformation = $e->getExtraInformation();
+			}
+			
+			foreach ($e->getTrace() as $traceEntry) {
```

Comment:
Add a big fat try-catch block around this. We don't want Exceptions being thrown within the Exception handler for some dumb reason.